### ISSUE TITLE
Non-negative output from scoring functions with cross-validation

### ISSUE THREAD

**Issue by Shammi270787** [juaml/julearn#60]
mean_absolute_error/mean_squared_error from sklearn.metrics returns negative of mean absolute error/ mean squared error with cross-validation which is very unintuitive.

**Comment by fraimondo** [comment]
Can you provide an example?

**Comment by samihamdan** [comment]
This is what @Shammi270787 is referring to (from sklearn documentation see link below): 

"
All scorer objects follow the convention that higher return values are better than lower return values. Thus metrics which measure the distance between the model and the data, like metrics.mean_squared_error, are available as *neg_mean_squared_error which return the negated value* of the metric.
"
https://scikit-learn.org/stable/modules/model_evaluation.html

We discussed this with @kaurao and want to keep the default behavior of sklearn. 
Maybe, we want to add a link to sklearn's docs in ours to refer to this pitfall. 
Else I would like to close this issue.
